claim_batch/services.py

```diff
@@ -534,6 +551,35 @@ def filter_item_or_service(base):
             .update(batch_run=created_run)
         logger.debug("do_process_batch updated claims *range* with batch run ref %s", updated_count)
 
+    capitation_payment_products = []
+    for svc_item in [ClaimItem, ClaimService]:
+        capitation_payment_products.extend(
+            svc_item.objects
+                    .filter(claim__status__in=[Claim.STATUS_PROCESSED, Claim.STATUS_VALUATED])
```

Comment:
Batch run is moving Claims from Processed to Valuated. Do this filter means you take also the Processed claims in Capitation report?

Reply:
Yes, Processed claims shouldn't be included. I changed filter to use only Valuated status.